config/initializers/cloudinary.rb

```diff
@@ -0,0 +1,6 @@
+Cloudinary.config do |config|
+  uri = URI.parse(ENV['CLOUDINARY_URL'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.